src/main/java/games/strategy/engine/lobby/server/userDB/MutedUsernameController.java

```diff
@@ -125,39 +123,4 @@ public long getUsernameUnmuteTime(final String username) {
     }
     return result;
   }
-
-  public List<String> getUsernamesThatAreStillMuted(final List<String> usernames) {
```

Comment:
This method was introduced in 5afd9f3, but there appears to never have been a call to it since.